inspire/config.py

```diff
@@ -225,7 +225,7 @@
     },
     "abstract": ["abstract.summary"],
     "collection": ["_collections"],
-    "collaboration": ["collaboration.collaboration"],
+    "collaboration": ["collaboration"],
```

Comment:
Same comment. Do we need this config when it is pointing to a first level of the JSON?